main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsStopTimeSchema.java

```diff
@@ -62,16 +63,21 @@ public interface GtfsStopTimeSchema extends GtfsEntity {
   int stopSequence();
 
   @CachedField
+  @UnusedValue
   String stopHeadsign();
 
+  @UnusedValue
   GtfsPickupDropOff pickupType();
 
+  @UnusedValue
   GtfsPickupDropOff dropOffType();
 
   @DefaultValue("1")
+  @UnusedValue
   GtfsContinuousPickupDropOff continuousPickup();
 
   @DefaultValue("1")
+  @UnusedValue
   GtfsContinuousPickupDropOff continuousDropOff();
 
   @NonNegative
```

Comment:
There was a problem with one of the datasets I used for testing (https://storage.googleapis.com/storage/v1/b/mdb-latest/o/ar-cordoba-mar-chiquita-srl-gtfs-1146.zip?alt=media).
In this data set, the shape_dist_traveled was empty except for one row that had a 0 in it.
Because of this a column was created with all values 0.
Maybe we can leverage the DefaultValue annotation and create a column only if a value in the file is different from the default?
Also the fact that the column existed created a bunch of decreasing_or_equal_stop_time_distance notices, because there was no increase of the distance. This would not have happened if hasShapeDistTraveled() returned false, but it returned true.